pkg/test/resource_creator_test.go

```diff
@@ -81,108 +83,115 @@ func TestGetWatchNamespace(t *testing.T) {
+				Global.WatchNamespace = ""
+				defer os.Unsetenv(WatchNamespaceEnv)
+			}()
+
+			ctx := NewContext(t)
+			got, err := ctx.GetWatchNamespace()
+			assertNoError(t, err)
+			if watchNamespace != got {
+				t.Errorf("expected watchNamespace: %s, got %s", watchNamespace, got)
+			}
+			operatorNamespace, err := ctx.GetOperatorNamespace()
+			assertNoError(t, err)
+			if len(operatorNamespace) <= 0 {
+				t.Errorf("expected non-empty operatorNamespace")
+			}
+			if watchNamespace == operatorNamespace {
+				t.Errorf("expected operator-Namespace: %v, to be different than watch-Namespace: %v",
+					operatorNamespace, watchNamespace)
+			}
+		})
+
+	t.Run("should return Watch Namespace (==WatchNamespace Env) even when WatchNamespace Env is set to \"\"",
+		func(t *testing.T) {
+			watchNamespace := ""
+			Global.WatchNamespace = watchNamespace
+			Global.OperatorNamespace = ""
+			os.Setenv(WatchNamespaceEnv, watchNamespace)
+			defer func() {
+				Global.WatchNamespace = ""
+				defer os.Unsetenv(WatchNamespaceEnv)
+			}()
+
+			ctx := NewContext(t)
+
+			got, err := ctx.GetWatchNamespace()
+			assertNoError(t, err)
+			if watchNamespace != got {
+				t.Errorf("expected watchNamespace: %s, got %s", watchNamespace, got)
+			}
+			operatorNamespace, err := ctx.GetOperatorNamespace()
+			assertNoError(t, err)
+			if len(operatorNamespace) <= 0 {
+				t.Errorf("expected non-empty operatorNamespace")
+			}
+			if watchNamespace == operatorNamespace {
+				t.Errorf("expected operator-Namespace: %v, to be different than watch-Namespace: %v",
+					operatorNamespace, watchNamespace)
+			}
+		})
+
+	t.Run("should return Watch Namespace and Operator Namespace when both Env are set through Env var",
+		func(t *testing.T) {
+			watchNamespace := "test-watch-namespace"
+			operatorNamespace := "test-operator-namespace"
+			Global.WatchNamespace = watchNamespace
+			Global.OperatorNamespace = operatorNamespace
+			os.Setenv(OperatorNamespaceEnv, operatorNamespace)
+			os.Setenv(WatchNamespaceEnv, watchNamespace)
+			defer func() {
+				Global.WatchNamespace = ""
+				Global.OperatorNamespace = ""
+				defer os.Unsetenv(OperatorNamespaceEnv)
+				defer os.Unsetenv(WatchNamespaceEnv)
+			}()
+
+			ctx := NewContext(t)
+			gotWatchNamespace, err := ctx.GetWatchNamespace()
+			assertNoError(t, err)
+			if watchNamespace != gotWatchNamespace {
+				t.Errorf("expected watchNamespace: %s, got %s", watchNamespace, gotWatchNamespace)
+			}
+
+			gotOperatorNamespace, err := ctx.GetOperatorNamespace()
+			assertNoError(t, err)
+			if len(gotOperatorNamespace) <= 0 {
+				t.Errorf("expected non-empty operatorNamespace")
+			}
+			if gotWatchNamespace == gotOperatorNamespace {
+				t.Errorf("expected operator-Namespace: %v, to be different than watch-Namespace: %v",
+					operatorNamespace, watchNamespace)
+			}
+			if gotOperatorNamespace != operatorNamespace {
+				t.Errorf("expected operatorNamespace: %s, got %s", operatorNamespace, gotOperatorNamespace)
+			}
+		})
```

Comment:
Could you confirm that you just change the indentation here?

Reply:
@camilamacedo86 the func is intented like this to preserve the 80 character lint rule.
i have kept the intendation consistent in other test cases.

Reply:
the indendation is automatically done by `go fmt`